packages/devtools_app/lib/src/shared/console/primitives/text_styles.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import '../../theme.dart';
+
+// Enum-like static classes are ok.
```

Comment:
why not continue the previous pattern of top level members imported with a prefix? In general if we were to use one pattern I would think we would migrate to that pattern not to this pattern that violates the lints we have enabled.

Reply:
Discussed offline and agreed enum-like static classes are more convenient than top level members.